Mathlib/Order/Nucleus.lean

```diff
@@ -136,4 +136,71 @@ instance : CompleteSemilatticeInf (Nucleus X) where
 
 instance : CompleteLattice (Nucleus X) := completeLatticeOfCompleteSemilatticeInf (Nucleus X)
 
+@[simp] theorem Min_apply (a b : Nucleus X) (x : X) : (a ⊓ b) x = a x ⊓ b x := by
```

Comment:
```suggestion
@[simp] theorem inf_apply (m n : Nucleus X) (x : X) : (a ⊓ b) x = a x ⊓ b x := by
```
since you've started using those letters higher up in the file